src/Sylius/Component/Core/spec/OrderProcessing/InventoryHandlerSpec.php

```diff
@@ -47,7 +50,7 @@ function it_implements_Sylius_inventory_handler_interface()
     }
 
     function it_creates_inventory_units_via_the_factory(
-        $inventoryUnitFactory,
+        InventoryUnitFactoryInterface $inventoryUnitFactory,
```

Comment:
We should use the interface also here https://github.com/pjedrzejewski/Sylius/blob/introduce-resource-factories/src/Sylius/Component/Core/spec/OrderProcessing/InventoryHandlerSpec.php#L36